### ISSUE TITLE
Partial Fourier tag

### ISSUE THREAD

**Issue by neurolabusc** [bids-standard/bep001#85]
The text for 01-magnetic-resonance-imaging-data.md defines PartialFourier as `The fraction of partial Fourier information collected. Corresponds to DICOM Tag 0018, 9081`. Be aware that [0018,9081](http://dicom.nema.org/medical/dicom/current/output/chtml/part03/sect_C.8.13.5.5.html) is an enumerated type of YES/NO not a fraction, e.g. `(0018,9081) CS [YES]`. Might be worth spelling this out, e.g. scan acquired with 5/8 Partial Fourier would report 0.625. I am not aware of a public tag that reports the desired value (for Siemens one can look at the CSA header).

**Comment by agahkarakuzu** [comment]
@neurolabusc great catch! Thank you for bringing this to our attention. I also could not find any generic tag for the partial Fourier fraction, that corresponds to the current description. 

I am not sure how widely this metadata is used by the existing data. Depending on the answer, backwards compatibility may prevent us from a complete redefinition. If anything, we can drop `Corresponds to DICOM Tag 0018, 9081` to avoid confusion and improve the description.

**Comment by neurolabusc** [comment]
Yes, I would simply drop the reference to the DICOM Tag. At the moment, dcm2niix reliably reports the PartialFourier for Siemens (ucPhasePartialFourier in CSA header). I would be happy to hear if anyone knows how to determine this for [Philips](https://github.com/rordenlab/dcm2niix/issues/377) or GE.
